packages/openapi-to-graphql/src/preprocessor.ts

```diff
@@ -1672,7 +1562,8 @@ function createOneOfUnion<TSource, TContext, TArgs>(
       memberSchema,
       isInputObjectType,
       data,
-      oas
+      oas,
+      def.links
```

Comment:
This line `def.links` allows OtG to trickle links from oneOf to its member schemas during initial creation of the data definitions, as opposed to collapsing links in existing data definitions from the previous comment.